openhgnn/dataset/LinkPredictionDataset.py

```diff
@@ -464,22 +466,41 @@ class KG_LinkPrediction(LinkPredictionDataset):
     """
     From `RGCN <https://arxiv.org/abs/1703.06103>`_, WN18 & FB15k face a data leakage.
     """
-    def __init__(self, dataset_name):
-        super(KG_LinkPrediction, self).__init__()
+    def __init__(self, dataset_name, *args, **kwargs):
+        super(KG_LinkPrediction, self).__init__(*args, **kwargs)
         if dataset_name in ['wn18', 'FB15k', 'FB15k-237']:
             dataset = load_data(dataset_name)
-            self.category = '_N'
-            self.num_rels = dataset.num_rels * 2
-            # include inverse edge
+            g = dataset[0]
+            self.num_rels = dataset.num_rels
```

Comment:
The reverse edges are not needed?

Reply:
1. params are fit for TransE in FB15k
2. no reverse edges needed